golang/cmd/aws_key_gen/README.md

```diff
@@ -0,0 +1,25 @@
+# Script to Create Tink Hybrid Keys using Envelope Encryption with AWS
+
+This script creates a [Tink Hybrid](https://github.com/google/tink/blob/master/docs/PRIMITIVES.md#hybrid-encryption) ([ECIES](https://en.wikipedia.org/wiki/Integrated_Encryption_Scheme) with AEAD) key and outputs the public key in plaintext to an output file, as well as the encrypted private key to another output file. The private key is encrypted using a key hosted on Amazon Web Services (AWS). This method is commonly referred to as [Envelope Encryption](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping).
+
+This script was inspired by the Medium post [Google Cloud KMS & Tink](https://medium.com/google-cloud/google-cloud-kms-tink-1e106156bb4e). Please read that post for more information.
+
+## Installation:
+
+```shell
+# Go get the script
+go get -u github.com/subscriptions-project/encryption/golang/cmd/aws_key_gen
+```
+
+## Example Usage:
+
+For aws credentials make sure you have `awscli` installed and you have configured it by running `aws configure` and make sure you have created credentials with default profile NOT NAMED profile.
```

Comment:
I don't fully understand "created credentials with default profile NOT NAMED profile."--could you explain what a default profile is? I think you can also leave off the "NOT NAMED profile" part.

Reply:
AWS has lots of cli tools available e.g. SAM cli, Amplify cli, CDK cli etc etc and all of them require Credentials and these credentials are stored with `awscli` (.aws folder) no matter which cli you use to add credential. This way you can share same credentials between multiple cli tools. e.g. same credentials are being automatically pulled with Tinkey.

Now each project requires different resource access. So different credentials can be generated per project basis. And you can store all credentials with [Named Profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-profiles). The difference being instead of `aws configure` you specify `aws configure --profile my-encryption`. So when its time to deploy resources to cloud then all cli tools ask which profile would you like to use for this project/deployment. 

I have my personal AWS account and I have full admin access to all resources with my default profile (the profile created with command `aws configure`).  So while testing I didn't explore named profiles. But other people may create credentials just for specific project with named profile. And their script won't work.

Reply:
Got it, thanks for explaining. How about just "...make sure you have created credentials with your default profile."?

Reply:
As the famous saying goes "don't tell but show"

So instead of telling them to use "default profile" I have already rephrased as follows.(its already added to PR)

> configured it by running `aws configure` NOT `aws configure --profile my-profile`